File: java/IEC61970/Dynamics/StandardModels/AsynchronousMachineDynamics/AsynchronousMachineDynamics.java
package IEC61970.Dynamics.StandardModels.AsynchronousMachineDynamics;

import IEC61970.Base.Wires.AsynchronousMachine;
import IEC61970.Dynamics.StandardModels.RotatingMachineDynamics;

/**
 * Asynchronous machine whose behaviour is described by reference to a standard
 * model expressed in either time constant reactance form or equivalent circuit
 * form <font color="#0f0f0f">or by definition of a user-defined model.</font>
 * 
 * <b>Parameter Notes:</b>
 * <ol>
 * 	<li>Asynchronous machine parameters such as <b>Xl, Xs</b> etc. are actually
 * used as inductances (L) in the model, but are commonly referred to as
 * reactances since, at nominal frequency, the per unit values are the same.
 * However, some references use the symbol L instead of X. </li>
 * </ol>
 * @author tsaxton
 * @version 1.0
 * @created 25-Dec-2023 8:31:54 PM
 */
public class AsynchronousMachineDynamics extends RotatingMachineDynamics {

	/**
	 * Asynchronous machine to which this asynchronous machine dynamics model applies.
	 */
	public AsynchronousMachine AsynchronousMachine;

	public AsynchronousMachineDynamics(){

	}
}//end AsynchronousMachineDynamics

File: py/IEC61970/Dynamics/StandardModels/AsynchronousMachineDynamics/AsynchronousMachineDynamics.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Sat Dec 30 21:57:45 2023
from IEC61970.Base.Wires.AsynchronousMachine import AsynchronousMachine
from IEC61970.Dynamics.StandardModels.RotatingMachineDynamics import RotatingMachineDynamics


class AsynchronousMachineDynamics(RotatingMachineDynamics):
    """ 
    Asynchronous machine whose behaviour is described by reference to a standard
    model expressed in either time constant reactance form or equivalent circuit form
    or by definition of a user-defined model.
    
    Parameter Notes:
    - Asynchronous machine parameters such as Xl, Xs etc. are actually
        used as inductances (L) in the model, but are commonly referred to as
        reactances since, at nominal frequency,
        the per-unit values are the same.
    - However, some references use the symbol L instead of X.
    
    @author: tsaxton
    @version: 1.0
    @created: 29-Dec-2023 11:24:17 PM
    """
    
    def __init__(self) -> None:
        super().__init__()
        self.asynchronous_machine: AsynchronousMachine = AsynchronousMachine()


